pkg/ddevapp/mutagen.go

```diff
@@ -589,6 +589,14 @@ func DownloadMutagenIfNeeded() error {
 			return err
 		}
 	}
+	// Also make sure the mutagen-agents.tar.gz is in place
+	agentsFile := filepath.Join(globalconfig.GetDDEVBinDir(), "mutagen-agents.tar.gz")
+	if !fileutil.FileExists(agentsFile) {
+		err = DownloadMutagen()
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
This is great! How about doing this with one if statement (incorporating it in the "or" above? You can calculate the agentsFile before the if, and add the !FileExists to the OR?

Reply:
Yes, makes sense!